web/setup-tests.ts

```diff
@@ -23,6 +23,14 @@ global.requestAnimationFrame = function (callback) {
 global.cancelAnimationFrame = function (id) {
   clearTimeout(id);
 };
+const actualComputedStyle = window.getComputedStyle;
```

Comment:
this can fix the failing test, getComputedStyle doesn't work as intended in tests .. though maybe there's a more idiomatic way to handle that